google/cloud/storage/client.cc

```diff
@@ -460,6 +461,7 @@ StatusOr<PolicyDocumentV4Result> Client::SignPolicyDocumentV4(
   if (!signed_blob) {
     return signed_blob.status();
   }
+  request.SetHost(Host());
```

Comment:
Seems like this should happen in the header file, where you create the `PolicyDocumentV4Request` parameter?